src/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -51,6 +52,9 @@
     <TargetingPackReference Include="mscorlib" />
     <TargetingPackReference Include="System.Data" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'net45'">
+    <Compile Include="System\Data\Common\DbDataReaderExtensions.Facade.cs" />
```

Comment:
NP: why not combine this into above group ? so only 1 item group contain this condition '$(TargetGroup)' == 'net45'" ?